tests/integration/bundles/kfp_latest_edge.yaml.j2

```diff
@@ -32,6 +32,7 @@ applications:
     charm: {{ kfp_schedwf }}
     resources: {{ kfp_schedwf_resources }}
     scale: 1
+    trust: true
```

Comment:
You should add `trust: true` in two places in this file. See https://github.com/canonical/kfp-operators/blob/6acdcf3ed17f5c99b208afa4380ef6454490d894/tests/integration/bundles/kfp_latest_edge.yaml.j2#L12

Reply:
+1